research/slim/nets/inception_v1_test.py

```diff
@@ -114,7 +114,7 @@ def testModelHasExpectedNumberOfParameters(self):
       inception.inception_v1_base(inputs)
     total_params, _ = slim.model_analyzer.analyze_vars(
         slim.get_model_variables())
-    self.assertAlmostEqual(5607184, total_params)
+    self.assertAlmostEqual(5988112, total_params)
```

Comment:
I've replaced the old value with the value returned by `slim.model_analyzer.analyze_vars()`.